.gitignore

```diff
@@ -26,3 +26,6 @@ doc/nrf/_doxygen/
 doc/nrf/_samples/
 doc/*_warnings.txt
 doc/*_doc.log
+
+ext/supl
```

Comment:
let's clean this up a bit and use `ext/lib/supl` please

Reply:
That must then be changed in the documentation as well.

Reply:
Ok,fixed